into/backends/sas.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import, division, print_function
+
+try:
+    import sas7bdat
+    from sas7bdat import SAS7BDAT
+except ImportError:
+    sas7bdat = None
+    SAS7BDAT = type(None)
+
+import datashape
+from datashape import discover, dshape, var
+from collections import Iterator
+import pandas as pd
+from ..append import append
+from ..convert import convert
+from ..resource import resource
+
+
+@resource.register('.+\.(sas7bdat)')
+def resource_sas(uri, **kwargs):
+    return SAS7BDAT(uri, **kwargs)
+
+
+@discover.register(SAS7BDAT)
+def discover_sas(f, **kwargs):
+    f.skip_header = True
+    ln = next(f.readlines())
+    ds = discover(ln)
+    cols = [col.name.decode("utf-8") for col in f.header.parent.columns]
+    types = [_type.strip() for _type in str(ds).strip("()").split(',')]
+    measure = ",".join(" " + col + ":" + _type for col, _type in zip(cols, types))
+    return var * dshape("{" + measure + "}")
```

Comment:
I was concerned about the way we discover types here.  After diving into the sas7bdat library a bit though I can't find a better way.  It looks like `f.column_types` would be good here but that seems to yield incorrect answers.  I can't find anything more robust than your method here.  I may end up changing out the string formatting for building up the DataShape programmatically.  I can do that in a separate commit though.